manifests/b/BitsumLLC/Coreprio/0.0.6.8/BitsumLLC.Coreprio.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: BitsumLLC.Coreprio

```

```suggestion
PackageIdentifier: BitSum.Coreprio
```
since there's already a `BitSum` folder in this repo.